### PR TITLE
[Bug] Fix module not found error and add linting with OpenMrs Guide

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-35: [Fix module not found error and add linting with OpenMrs Guide](https://issues.openmrs.org/browse/PDMO-35)

## SUMMARY:
Fix module not found error and add linting with OpenMrs Guide

### CODE REVIEW

**Comment by jomadoye** [comment]
Good job @andela-cuchendu, looks good to me.

File: .eslintrc [hunk 1]
```diff
@@ -1,74 +0,0 @@
...
-    "react/no-unknown-property": 1,
-    "react/prefer-es6-class": 1,
-    "react/prop-types": 1,
-    "react/react-in-jsx-scope": 1,
-    "import/extensions": 1,
-    "react/self-closing-comp": 1,
-    "react/sort-comp": 1,
-    "react/jsx-wrap-multilines": 1
-  }
-}
```
Comment (andela-jomadoye) [inline .eslintrc]: Slight EOL error here.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        "react/no-unknown-property": 1,
+        "react/prefer-es6-class": 1,
+        "react/prop-types": 1,
+        "react/react-in-jsx-scope": 1,
+        "import/extensions": 1,
+        "react/self-closing-comp": 1,
+        "react/sort-comp": 1,
+        "react/jsx-wrap-multilines": 1
+    }
+};
```
Comment (jomadoye) [inline .eslintrc.js]: EOL error here

File: .eslintrc [hunk 3]
```diff
@@ -1,74 +0,0 @@
-{
```
Comment (andela-jomadoye) [inline .eslintrc]: It would be nice to set a maximum length per line rule here.

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+module.exports = {
```
Comment (andela-jomadoye) [inline .eslintrc.js]: It would be nice to set a rule for the maximum length allowed per line in the eslint file.

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+          "jsx": true
+        }
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
```
Comment (jomadoye) [inline .eslintrc.js:20]: It would be nice to set a rule for the maximum length allowed per line in the eslint file.

**Comment by chibujax** [comment]
Thank you @andela-jomadoye for the observations, new line added and max-len added as well, thank you 😄

File: .eslintrc.js (hunk 2 above)
Comment (chibujax) [reply .eslintrc.js]: Added

File: .eslintrc.js (hunk 5 above)
Comment (chibujax) [reply .eslintrc.js:20]: Done

**Comment by kingisaac95** [comment]
This looks good to me. Good job @andela-cuchendu

**Pull request merged**